hack/update-all.sh

```diff
@@ -14,77 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A single script that runs a predefined set of update-* scripts, as they often go together.
+# This script is a vestigial redirection.  Please do not add "real" logic.
```

Comment:
might be worth adding a comment indicating where the update-all wrapper went (`hack/make-rules/update.sh`)